.github/.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
```

Comment:
this file should be located on `.github/workflows` folder, current path starts with `.github/.github/workflows`, it should be  `.github/workflows`

Reply:
That is why It did not check in GITHUB, thank you!